src/main/java/org/sagebionetworks/bridge/adherence/WeeklyAdherenceReportWorkerProcessor.java

```diff
@@ -150,8 +151,16 @@ private void process(WeeklyAdherenceReportRequest request) throws Exception {
                     }
                     
                     LOG.info("Caching report for user " + summary.getId() + " in study " + studyId);
-                    WeeklyAdherenceReport report = workersApi.getWeeklyAdherenceReportForWorker(
-                            app.getIdentifier(), studyId, summary.getId()).execute().body();
+                    WeeklyAdherenceReport report = null;
+                    try {
+                        report = workersApi.getWeeklyAdherenceReportForWorker(
+                                app.getIdentifier(), studyId, summary.getId()).execute().body();
+                    } catch (BridgeSDKException e) {
+                        // Catching this specific error to prevent a single account issue from
+                        // stopping the remaining reports
+                        LOG.error("Error while processing adherence job", e);
```

Comment:
Minor: I would include app, study, and user ID in the error message. This will save us a step when looking at the logs.

Reply:
That's a good call! I'll add that before merging.